### PR TITLE
Make color map, status map match the available build status

### PR DESCRIPTION
**Pull request by jweingarten**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,20 +5,31 @@ const slacker = require('./slack');
...
+    BLOCKED: '#ccc', // Using 'sd-light-gray' from https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
+    UNSTABLE: '#ffd333', // Using 'sd-unstable' from https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
+    COLLAPSED: '#f2f2f2', // New color. Light grey.
+    FROZEN: '#acd9ff' // Using 'sd-frozen' from https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
 };
 const STATUSES_MAP = {
-    SUCCESS: ':sunny:',
-    FAILURE: ':umbrella:',
     ABORTED: ':cloud:',
+    CREATED: ':progress:',
```
Comment (adong) [inline index.js]: The following `emoji` or slack emoticons does NOT exist for standard slack.

```
:progress: 
```

I typed it in the SD Slack, and its not available, and also searched in 
https://screwdriver-cd.slack.com/customize/emoji?utm_source=in-prod&utm_medium=inprod-customize_link-slack_menu-click

To see if its standard or customized.

Maybe that can be changed for something more standard like `:clock11:`? 
image

Comment (jweingarten) [reply index.js]: :progress: works in my Slack. I tested them all

Comment (jweingarten) [reply index.js]: But I can change it ...

Comment (jweingarten) [reply index.js]: Done

**Pull request merged**